app/assets/stylesheets/layout_components/siteheader.scss

```diff
@@ -95,15 +95,17 @@
     padding: 0 $base-padding;
   }
 
+  a{
```

Comment:
I agree, I went into ama_layout to do this but way the link and image are built I'm not sure how to do it, so i'll ask a dev for help the next time I get a chance to work on this.